en/Useful-Websites-to-Learn-How-to-Code-Quickly.md

```diff
@@ -0,0 +1,95 @@
+* zh translation:  [wangguo](http://wangguo.iteye.com/)
+
+The Internet is a rapidly evolving field, where technologies constantly being updated, like from HTML to HTML5, from CSS to CSS3, from JavaScript to JavaScript frameworks, etc. If you can't keep up with the speed, you will be out of game. Therefore, a quick grasp of a language or a technology can be very beneficial. 
+
+This article provides you 9 practical online tutorials, some in interactive format, and some in comprehensive guides and visualizations to help you quickly master a language. They share a common purpose, that is, make your learning more fun, and make it easier for you to master it.
+
+##  [LifeHacker Learn to code](http://www.lifehacker.com.au/2011/02/learn-to-code-the-full-beginners-guide/)
+
+Language: JavaScript 
+4 lessons plus best practices/resources to teach you how to program using Javascript. 
+
+Each lesson has a video and detailed articles to make sure you understand them.  
+
+![LifeHacker](images/9-useful-LifeHacker.jpg)
+
+##  [CodeCademy](http://www.codecademy.com/)
+
+Language: JavaScript 
+
+This is an interactive form of instruction that gives you a bit of a grip on JavaScript. 
+
+For each course you complete, you get some achievement points and badges that will inspire your interest in learning. 
+
+![](images/9-useful-CodeCademy.jpg)
+
+
+## [TeamTreeHouse](http://teamtreehouse.com/)
+
+
+Languages: CSS, CSS3, HTML, HTML5, JavaScript, Basic Programming, and iOS Development 
+
+This is a paid member online learning service that covers topics such as Web design and development, as well as iOS development, which is now very popular. 
+
+![](images/9-useful-TeamTreeHouse.jpg)
+
+## [RubyMonk](http://rubymonk.com/)
+
+Language: Ruby RubyMonk is an interactive learning platform. 
+
+You can learn Ruby through courses, problem solving, or related articles. 
+
+![](images/9-useful-rubyMonk.jpg)
+
+## [Hackety](https://github.com/hacketyhack/hacketyhack)
+
+Language: Ruby 
+
+Hackety will teach you the absolute basics of programming, even if you don't have any programming experience. 
+
+![](images/9-useful-hackety.jpg)
+
+## [jQuery Air](http://jqueryair.com/)
+
+
+Language: JavaScript jQuery is the most famous JavaScript framework. Now you can learn jQuery directly in your browser. 
+
+The process of learning jQuery should be fun, and jQuery Air does it through a number of practical ways. 
+
+![](images/9-useful-jqueryAir.jpg)
+
+
+## [CodingBat](http://codingbat.com/)
+
+Language: Java, Python 
+
+Learn about Java and Python by solving sample problems and online coding practices. 
+
+![](images/9-useful-codingBat.jpg)
+
+## [PHP Know How](http://www.phpknowhow.com/)
+
+Language: PHP 
+
+This is not an interactive way to learn, but it has a good written tutorial that contains a wealth of examples and guides to teach you the basics of PHP and MySQL. 
```

Comment:
This is not an interactive way to learn, but it has a well written tutorial that contains a wealth of examples and guides to teach you the basics of PHP and MySQL.